src/Tasks.UnitTests/Copy_Tests.cs

```diff
@@ -84,6 +84,38 @@ public void Dispose()
             Copy.RefreshInternalEnvironmentValues();
         }
 
+        [Fact]
+        [SkipOnPlatform(TestPlatforms.Windows, "The error message derived directly from the hardlink creation failure is not provided by MSBuild, so including what it actually logs would be artificial.")]
```

Comment:
This doesn't make sense to me. Isn't it unix-only because we have an OS-specific p/invoke?

Reply:
We have matching p/invokes per OS, so one could suppose we could easily craft the appropriate error message a la:
if (NativeMethodsShared.IsWindows)
{
    errorMessage = ...
}
else
{
    errorMessage = ...
}

But that doesn't work because the windows version isn't from MSBuild and isn't preserved in any particularly relevant form. We can't craft an appropriate error from what's in the test, and I don't think it's worth it to try to plumb the error message back out to some output variable.